.env

```diff
@@ -6,4 +6,4 @@ REACT_APP_CAMPAIGN_MANAGER_ADDRESS=0xCf7Ed3AccA5a467e9e704C703E8D87F634fB0Fc9
 REACT_APP_DAOSEEDER_FACTORY_ADDRESS=0x5FbDB2315678afecb367f032d93F642f64180aa3
 REACT_APP_STAGE_ADDRESS=0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512
 REACT_APP_ETHEREUM_BLOCK_TIME=12
-REACT_APP_NETWORK_CURRECNY=ETH
\ No newline at end of file
+REACT_APP_NETWORK_CURRENCY=ETH
```

Comment:
We still need to detect the currency name from the blockchain or the metamask or wagmi.